packages/test-helpers/src/aragon-os/apps.js

```diff
@@ -1,13 +1,9 @@
-const { decodeEvents } = require('../decoding')
-const { getArtifacts } = require('../config')
+const { getEvents } = require('../events')
 
-function getInstalledApps(receipt, appIds, ctx) {
+function getInstalledApps(receipt, appIds) {
   appIds = Array.isArray(appIds) ? appIds : appIds ? [appIds] : null
 
-  const artifacts = getArtifacts(ctx)
-  const Kernel = artifacts.require('Kernel')
-
-  return decodeEvents(receipt, Kernel.abi, 'NewAppProxy')
+  return getEvents(receipt, 'NewAppProxy')
```

Comment:
I think we should still input the specific ABI here.

Maybe you did a call (e.g. to a template) where these weren't on the contract you directly called (and so the truffle contract doesn't know about the ABI).